index.html

```diff
@@ -10,63 +10,71 @@
 </head>
 
 <body>
-
-  <nav></nav>
-
-  <h1>Front-End Tools</h1>
-
+  <h1 class="title-tech">Front-End Tools</h1>
```

Comment:
You shouldnt have more than one `h1` per scree, so I think you should use an `h2` here